andela/asgard-cfh
# #157801289 Convert Grunt to Gulp taskrunner

#### What does this PR do?
Convert Grunt to Gulp on the andela/asgard-cfh repository

#### Description of Task to be completed?
- Add gulpfile.babel.js file to the repository
- Delete grunt.js file
- Add .babelrc file
- Install all dependencies and plugins for gulp.

#### How should this be manually tested?
- `npm start`  To start the server with gulp.
- Gulp should run nodemon and run task

#### What are the relevant pivotal tracker stories?
#157801289
